docs/dev/check_references.md

```diff
@@ -99,6 +111,76 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
+| `assets[integration[metrics[check]]]`                | String or List of String                                                | Mandatory                                  | A string or list representing a metric that this integration always emits on every run.                                                                                                                                                                    |   |
+| `assets[integration[metrics[metadata_path]]]`        | String                                                                  | Mandatory                                  | Relative path to where the metrics metadata lives in relation to this manifest.                                                                                                                                                                            |   |
+| `assets[integration[metrics[prefix]]]`               | String                                                                  | Mandatory                                  | The prefix for metrics emitted by this integration.                                                                                                                                                                                                        |   |
+| `assets[integration[service_checks]]`                | Object                                                                  | Mandatory, but can be `{ }`                | Information about service checks emitted by this integration.                                                                                                                                                                                              |   |
+| `assets[integration[service_checks[metadata_path]]]` | String                                                                  | Mandatory                                  | Relative path to where the service check metadata lives in relation to this manifest.                                                                                                                                                                      |   |
+| `assets[integration[supported_os]]`                  | List of String (Enum)                                                   | Mandatory, can be `[ ]`                    | The available operating systems this integration can be used on.                                                                                                                                                                                           |   |
```

Comment:
For manifest V2, supported_os is now part of the `classifier_tags`. This row can be removed